gen3-integration-tests/services/fence.py

```diff
@@ -1,27 +1,40 @@
 import json
-import os
 import pytest
 import requests
+import base64
 
 from utils.misc import retry
 
 from utils import logger
+from pages.login import LoginPage
 from gen3.auth import Gen3Auth
+from playwright.sync_api import Page
+from utils.test_execution import screenshot
 
 
 class Fence(object):
     def __init__(self):
         self.BASE_URL = f"{pytest.root_url}/user"
-        self.API_CREDENTIALS_ENDPOINT = f"{self.BASE_URL}/credentials/api"
-        self.OAUTH_TOKEN_ENDPOINT = f"{self.BASE_URL}/oauth2/token"
-        self.DATA_UPLOAD_ENDPOINT = f"{self.BASE_URL}/data/upload"
-        self.DATA_ENDPOINT = f"{self.BASE_URL}/data"
-        self.USER_ENDPOINT = f"{self.BASE_URL}/user"
+        self.API_CREDENTIALS_ENDPOINT = "/credentials/api"
```

Comment:
Why is this changing? The previous method seems more efficient. In the previous method you could use a string manipulation to remove the BASE_URL in the few cases where it should be stripped out.

Reply:
The endpoint here is being used without self.BASE_URL since the calls made using the gen3sdk already uses the endpoint when invoking the Gen3Auth object. We then just need to provide the endpoint without self.BASE_URL to perform a curl operation. Since only get request is working properly in gen3auth curl function we have a mix of both gen3sdk and python requests to perform api call, we removed it to access it easily in both ways rather than creating separate variables.